Financial Assistance (#7)
PyconUK/2022.pyconuk.org: src/financial-assistance.md

---
layout: default
title: Financial Assistance
---

Our financial assistance programme exists to support attendees who might otherwise find it difficult to attend the conference.
We have a budget set aside for this purpose.

> The deadline for applications is 17:00 (Europe/London) on Friday, 17th June 2022.

Reviewer: Would suggest adding an are we meeting yet link to the time to make it easier for folks to track timezones

The organisers will not be able to accept applications after that date.

The financial assistance programme prioritises speakers.
We also use it to support attendees from groups who are under-represented in our community and at our events,
as part of our efforts to make PyCon UK as inclusive and diverse as possible.

Sometimes people feel reluctant to apply,
for all kinds of reasons.
For example, they feel might embarrassed to ask for help,
or they doubt that they are the right kind of person,
or they think that they don't deserve it,
or that other people are more deserving.
We hope that you won't feel like this, and we will be glad to receive your request for support.

We welcome applications from all attendees, and strongly encourage you to apply if it would make it easier for you to attend.
Many of your fellow attendees will be doing the same, including many speakers and even members of the conference organisation team.

> This year we are currently only able to offer financial assistance in the form of free tickets.

Reviewer: ```suggestion
Please note that this year we will only be able to offer financial assistance in the form of free tickets.
```
